guest.php

```diff
@@ -463,8 +463,12 @@ private function httpExit(int $code = 404)
 
     private function __construct()
     {
-        $this->client = filter_var($_GET['client_id'], FILTER_SANITIZE_STRING);
-        $this->refId = filter_var($_GET['ref_id'], FILTER_SANITIZE_NUMBER_INT);
+        $this->client = isset($_GET['client_id']) ? filter_var($_GET['client_id'], FILTER_SANITIZE_STRING) : null;
+        $this->refId = isset($_GET['ref_id']) ? filter_var($_GET['ref_id'], FILTER_SANITIZE_NUMBER_INT) : 0;
+        if (is_null($this->client)) {
```

Comment:
Is already exist in `initIlias` :

https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin/blob/83bd58fb289f52b17da3272e5aa0489347dba7b9/guest.php#L50-L54